### ISSUE TITLE
Changing page language shows modal in current language

### ISSUE THREAD

**Issue by davenewham** [datenanfragen/website#694]
i.e if on the german website and I try to change the language, I will be shown a modal in German warning me 
![image](https://user-images.githubusercontent.com/15076177/134638120-cb72cc11-2dfe-4172-abc0-9a9f0ecff0ce.png).

I think it would be more expected to have both target selected language appear, and current page language. Perhaps the buttons should be in the target language?

**Comment by TomerPacific** [comment]
Can I help out with this issue?

**Comment by davenewham** [comment]
Hi @TomerPacific, I've been working on some changes to this I18NWidget and suggesting other languages on the PR #615. This PR has now changed the behaviour, to where the target language is shown in the modal (instead of the current language). If you wanted to help out, you could try and figure out how to display both current and target language? Maybe with a HR element between them?

**Comment by TomerPacific** [comment]
@davenewham , I've looked over the PR. Would it be best that I branch out of the PR's branch and opened a new PR for that?

**Comment by davenewham** [comment]
Yes, I think so. Thanks for helping 😊

**Comment by baltpeter** [comment]
The PR is now merged, so you can also work straight from master if you haven't started already. :)

**Comment by TomerPacific** [comment]
@baltpeter , should I open a new issue then or would you like to keep this one open?

**Comment by baltpeter** [comment]
@TomerPacific I see no reason to open a new issue, let's use this one. I can assign you if you want.

**Comment by TomerPacific** [comment]
@baltpeter, sure let's do that. Just wanted to make sure I am aligned with your working style.

**Comment by TomerPacific** [comment]
Just to make sure I understand what needs to be done, you would like to display in the modal, the current language of the website and also the target language selected?

If this is so, would you like that the different versions of the text appear inside the buttons or somewhere else? I.E. at the top of the modal?
![website](https://user-images.githubusercontent.com/23402988/135854956-1881bbfc-b063-4327-bf74-1eaf7805a39e.jpg)

**Comment by baltpeter** [comment]
> just to make sure I understand what needs to be done, you would like to display in the modal, the current language of the website and also the target language selected?

Yes, exactly. Otherwise, a user who accidentally selects the wrong language might not find their way back.

> If this is so, would you like that the different versions of the text appear inside the buttons or somewhere else? I.E. at the top of the modal?

Not sure. As @davenewham [said](https://github.com/datenanfragen/website/issues/694#issuecomment-932900979), we haven't figured out a design for that yet. That's also part of the scope of this issue.

**Comment by davenewham** [comment]
![image](https://user-images.githubusercontent.com/15076177/135885176-e00fcf00-6ebf-4b14-a419-4451e3eb5e1b.png)
![image](https://user-images.githubusercontent.com/15076177/135885331-e1de58e0-86fc-44ff-84ab-15eed4a3ca44.png)

These are some of the ideas I had on how I was going to implement it - I particularly like the first one (similar to how Firefox does it) 
![image](https://user-images.githubusercontent.com/15076177/135885464-a8ee429a-3ed7-4334-a677-15f9742ab94b.png)

Obviously these are just my ideas, you are welcome to come up with something better :)

**Comment by zner0L** [comment]
Maybe hide the other language in a `<detail>` element?

**Comment by TomerPacific** [comment]
@baltpeter, just wanted to update that I am having some issues with setting up the local environment. I am working on a Windows machine and am following the development guide. It seems that running the deploy.sh script takes a really long time. Don't know if that is the expected behavior.
